src/modules/repository/logs_repository.js

```diff
@@ -5,22 +5,27 @@ module.exports = class LogsRepository {
     this.db = db;
   }
 
-  getLatestLogs(excludes = ['debug'], limit = 200) {
+  getLatestLogs(filters = { logExcludeLevels: ['debug'], logTxtFilter: '' }, limit = 200) {
     return new Promise(resolve => {
-      let sql = `SELECT * from logs order by created_at DESC LIMIT ${limit}`;
-
       const parameters = {};
+      let sql = `SELECT * from logs WHERE 1=1`;
```

Comment:
why do you need `WHERE 1=1`here?

Reply:
This is to be able later add where conditions and do not care if it is first condition or not.

A more elegant solution would be to push all conditions to array and then use conditions.join(' AND '). I used to build queries such way.

But please wait with this pull request, I'm working on [datatables](https://newcodingera.com/datatables-server-side-processing-using-nodejs-mysql/) solution, which will auto generate sql query based on filters. It will also add pagination, filtering, sorting as well. We could also use it in other pages if needed.